lib/Team.js

```diff
@@ -0,0 +1,172 @@
+/**
+ * @file
+ * @copyright  2013 Michael Aufreiter (Development Seed) and 2016 Yahoo Inc.
```

Comment:
This probably wants to be you (or whomever you've signed your copyright over to), neither Yahoo nor Michael Aufreiter/Development Seed wrote this code.